apps/optimizely/src/ConnectButton.spec.jsx

```diff
@@ -2,20 +2,15 @@ import React from 'react';
 import { cleanup, render, fireEvent, configure } from '@testing-library/react';
 
 import ConnectButton from '../src/ConnectButton';
+import { vi } from 'vitest';
 
 configure({ testIdAttribute: 'data-test-id' });
 
 describe('ConnectButton', () => {
   afterEach(cleanup);
 
-  it('should match snapshot', () => {
```

Comment:
I just removed this snapshot since it didn't seem very useful, and I had the problem with the snapshot constantly changing due to the forma button classNames. It wasn't testing anything specific. The other test checks for the test id that should be found if this connect button renders successfully and I think that's sufficient?